Documentation/ApiOverview/CommandControllers/Index.rst

```diff
@@ -97,6 +97,16 @@ Create a custom command
 See the :ref:`Tutorial: Create a console command <console-command-tutorial>`
 for details on how to create commands.
 
+DataHandler usage
+=================
+Using the :ref:`DataHandler <tce-database-basics>` in a cli command will
+fail with the following error:
+
+ [1.2.1]: Attempt to modify table "pages" without permission
+
+See :ref:`Using the DataHandler in a Symfony command <dataHandler-cli-command>`
```

Comment:
Maybe only insert the ref without a title, so that it will always use the title of the target doc?